rust/libnewsboat/src/utils.rs

```diff
@@ -51,6 +51,22 @@ pub fn get_random_value(max: u32) -> u32 {
    rand::random::<u32>() % max
 }
 
+extern crate urlencoding;
```

Comment:
Please move this line to the beginning of the file, as we discussed in your previous PR.

Reply:
Thanks